### ISSUE TITLE
This looks close!

### ISSUE THREAD

**Issue by av8ta** [sim51/graphology-gremlin#1]
Hi this project looks great and it looks like it's well advanced! I mean the functions you have in https://github.com/sim51/graphology-gremlin/blob/main/src/traversal/graphTraversal.ts look comprehensive to me. How much more work is there to get basic queries, at least, working?

**Comment by sim51** [comment]
All functions in this files are already implemented, but it's just a subset of the gremlin language.

With that you can already traverse your graph, and do some checks on your nodes/edges (item with a prop, a label, ...).  
So  basic queries should work.

But the gremlin `where` predicate is not there yet. I need to find a way to implement it with a good typescript support.

Moreover, I need to make a realistic fixtures.
I took a look at the gremlin compliance test framework, but it's too much of work.  

What can be cool, is to have a realistic dataset in graphology, and a list of gremlins queries with their result.

**Comment by av8ta** [comment]
Yeah I think to be minimally useful you need `where` `and` `or` functionality (I don't know gremlin btw, just assume that's how it works!).
